dss/util/auth/authorize.py

```diff
@@ -107,5 +107,20 @@ def _assert_authorized_email(self, emails):
         return
 
 
-class Authorize(TokenGroupMixin, TokenEmailMixin):
+class AdminStatusMixin(TokenGroupMixin, TokenEmailMixin):
+    @property
+    def admin_emails(self):
```

Comment:
this might belong in Config.

Reply:
Ah, thanks. I knew there was a better way to do this...

Reply:
Sorry, misunderstood this to mean there was already a method in Config that did this, but it looks like there is not yet. You're right, that would be a better place for this bit of code.

Reply:
Fixed with #132